layouts/page/nx-cugraph.html

```diff
@@ -0,0 +1,325 @@
+          <br>
+          
+          <p>From the shell, set an environment variable: </p>
+          <code class="code-sample mt-3">
+            NX_CUGRAPH_AUTOCONFIG=True<br>
+            python my_networkx_script.py
+          </code>
+
+          <br>
+
+          <p>From a notebook, use %env before the networkx import:</p>
+          <code class="code-sample mt-3">
+            %env NX_CUGRAPH_AUTOCONFIG=True<br>
+            import networkx
+          </code>
+
+          <br>
+
+          <p>See the <a href="https://networkx.org/documentation/stable/tutorial.html#using-networkx-backends" target="_blank"> NetworkX documentation</a> for additional ways to enable and use backends.</p>
+
+          <br>
+
+          <p>Learn more about these benchmark results in the <a href="https://developer.nvidia.com/blog/networkx-introduces-zero-code-change-acceleration-using-nvidia-cugraph/" target="_blank"> Announcement Blog</a> and <a href="https://docs.rapids.ai/api/cugraph/stable/nx_cugraph/" target="_blank"> nx-cugraph documentation</a>.</p>
+        </div>
+        <div class="col-md-4 py-3">
+
+          <canvas id="perfchartA" width="400" height="450"></canvas>
+          <p class="fs-8 fw-lighter text-center">* Dataset: cit-Patents | 3.7M Nodes, 16.5M Edges<br>GPU: NVIDIA A100, 80 GB<br>CPU: Intel Xeon W9-3495X (Sapphire Rapids)</p>
+       
+        </div>
+        <div class="col-md-4 py-3">
+
+          <canvas id="perfchartB" width="400" height="450"></canvas>
+          <p class="fs-8 fw-lighter text-center">* Dataset: soc-livejournal1 | 4.8M Nodes, 69M Edges<br>GPU: NVIDIA A100, 80 GB<br>CPU: Intel Xeon W9-3495X (Sapphire Rapids); Betweenness = Betweenness Centrality with k=100</p>
+        </div>
+      </div>
+    </div>
+</section>
+
+
+<div class="top-slant-container">
+  <div class="top-slant-up bg-rapids-gray"></div>
+</div>
+
+<section class="row section bg-rapids-gray padding-bottom-slant">
+  <div class="col">
+    <div class="container">
+      
+      <h1 class="mt-10 mb-4"> How It Works </h1>
+
+      <div class="row">
+        <div class="col-md-6 py-3">
+          <h2><i class="fa-regular fa-light fa-gear-code"></i> Under the Hood </h2>
+          <p>NetworkX has the ability to dispatch function calls to separately-installed third-party backends. These NetworkX backends let users experience improved performance and/or additional functionality without changing their NetworkX Python code. Examples include backends that provide algorithm acceleration using GPUs, parallel processing, graph database integration, and more.</p>
+          
+          <br>
+          
+          <p>While NetworkX is a pure-Python implementation, backends may be written to use other libraries and even specialized hardware. As such, `nx-cugraph` is a NetworkX backend that uses RAPIDS cuGraph and NVIDIA GPUs to significantly improve NetworkX performance.</p>
```

Comment:
Code formatting with backticks is Markdown syntax and is not supported in HTML. Let's use plain text here instead.
```suggestion
          <p>While NetworkX is a pure-Python implementation, backends may be written to use other libraries and even specialized hardware. As such, nx-cugraph is a NetworkX backend that uses RAPIDS cuGraph and NVIDIA GPUs to significantly improve NetworkX performance.</p>
```